blog/electron-35-0.md

```diff
@@ -0,0 +1,157 @@
+in favor of `session.serviceWorkers.getInfoFromVersionID(versionId)`. This was
+changed to make it more clear which object is returned with the introduction
+of the `session.serviceWorkers.getWorkerFromVersionID(versionId)` API.
+
+```js
+// Deprecated
+session.serviceWorkers.fromVersionID(versionId);
+
+// Replace with
+session.serviceWorkers.getInfoFromVersionID(versionId);
+```
+
+### Deprecated: `setPreloads`, `getPreloads` on `Session`
+
+`registerPreloadScript`, `unregisterPreloadScript`, and `getPreloadScripts` are introduced as a
+replacement for the deprecated methods. These new APIs allow third-party libraries to register
+preload scripts without replacing existing scripts. Also, the new `type` option allows for
+additional preload targets beyond `frame`.
+
+```js
+// Deprecated
+session.setPreloads([path.join(__dirname, 'preload.js')]);
+
+// Replace with:
+session.registerPreloadScript({
+  type: 'frame',
+  id: 'app-preload',
+  filePath: path.join(__dirname, 'preload.js'),
+});
+```
+
+### Deprecated: `level`, `message`, `line`, and `sourceId` arguments in `console-message` event on `WebContents`
+
+The `console-message` event on `WebContents` has been updated to provide details on the `Event`
+argument.
+
+```js
+// Deprecated
+webContents.on(
+  'console-message',
+  (event, level, message, line, sourceId) => {},
+);
+
+// Replace with:
+webContents.on(
+  'console-message',
+  ({ level, message, lineNumber, sourceId, frame }) => {},
+);
+```
+
+Additionally, `level` is now a string with possible values of `info`, `warning`, `error`, and `debug`.
+
+### Behavior Changed: `urls` property of `WebRequestFilter`.
+
+Previously, an empty urls array was interpreted as including all URLs. To explicitly include all URLs, developers should now use the `<all_urls>` pattern, which is a [designated URL pattern](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#all_urls) that matches every possible URL. This change clarifies the intent and ensures more predictable behavior.
+
+```js
+// Deprecated
+const deprecatedFilter = {
+  urls: [],
+};
+
+// Replace with
+const newFilter = {
+  urls: ['<all_urls>'],
+};
+```
+
+### Deprecated: `systemPreferences.isAeroGlassEnabled()`
+
+The `systemPreferences.isAeroGlassEnabled()` function has been deprecated without replacement.
+It has been always returning `true` since Electron 23, which only supports Windows 10+, where DWM composition can no longer be disabled.
+
+https://learn.microsoft.com/en-us/windows/win32/dwm/composition-ovw#disabling-dwm-composition-windows7-and-earlier
+
+## End of Support for 32.x.y
+
+Electron 32.x.y has reached end-of-support as per the project's [support policy](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+| E35 (Apr'25) | E36 (Jun'25) | E37 (Aug'25) |
```

Comment:
These months don't match what's on the [Electron Releases](https://www.electronjs.org/docs/latest/tutorial/electron-timelines) page.

In particular, Electron 35 should probably be Mar'25. Electron 36 is listed as April on the Electron Releases page.

Reply:
```suggestion
| E35 (Mar'25) | E36 (Apr'25) | E37 (Aug'25) |
```

This would bring us in line with the Releases page. However, the August part still seems fishy to me.

Reply:
fixed, with e37 in jun